README.md

```diff
@@ -1 +1,966 @@
+
+##### phone_store_service.bal
+```ballerina
+import ballerinax/docker;
+// Other imports
+
+// Type definition for a phone order
+
+json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+@docker:Config {
+    registry:"ballerina.guides.io",
+    name:"bookstore_service",
+    tag:"v1.0"
+}
+
+
+@docker:Expose{}
+endpoint http:Listener listener {
+    port:9090
+};
+
+
+@http:ServiceConfig {basePath:"/phonestore"}
+service<http:Service> phone_store_service bind listener {
+``` 
+Similar to the `phone_store_service.bal`, We define the `@docker:Config` and `@docker:Expose {}` in  `phone_order_delivery_service` for Docker deployment.
+
+- `@docker:Config` annotation is used to provide the basic Docker image configurations for the sample.`@docker:Expose {}` is used to expose the port. 
+
+- Now you can build a Ballerina executable archive (.balx) of the service that we developed above, using the following command. This will also create the corresponding Docker image using the Docker annotations that you have configured above. Navigate to `message_construction_patterns/guide` and run the following command.  
+  
+```
+   $ballerina build 
+  
+   ./target/phone_store.balx
+        @docker                  - complete 3/3 
+
+        Run following command to start docker container:
+        docker run -d -p 9090:9090 ballerina.guides.io/phone_store_service:v1.0
+
+    ./target/phone_order_delivery_service.balx
+        @docker                  - complete 3/3 
+
+        Run following command to start docker container:
+        docker run -d -p 9091:9091 ballerina.guides.io/phone_order_delivery_service:v1.0
+
+```
+
+- Once you successfully build the Docker image, you can run it with the `` docker run`` command that is shown in the previous step.  
+
+```bash
+  docker run -d -p 9090:9090 ballerina.guides.io/phone_store_service:v1.0
+  docker run -d -p 9091:9091 ballerina.guides.io/phone_order_delivery_service:v1.0  
+```
+
+   Here we run the Docker image with flag`` -p <host_port>:<container_port>`` so that we use the host port 9090 and the container port 9090. Therefore you can access the service through the host port. 
+
+- Verify docker container is running with the use of `` $ docker ps``. The status of the Docker container should be shown as 'Up'. 
+
+- You can access the service using the same curl commands that we've used above.
+```bash
+    curl -v -X POST -d \
+   '{"Name":"John", "Address":"20, Palm Grove, Colombo, Sri Lanka", 
+   "ContactNumber":"+94718930874", "PhoneName":"Apple:190000"}' \
+   "http://localhost:9090/phonestore/placeOrder" -H "Content-Type:application/json"
+```
+### Deploying on Kubernetes
```

Comment:
The instructions for Linux users to use minikube is missing in this section. See https://github.com/ballerina-guides/restful-service/blob/master/README.md#deploying-on-kubernetes